src/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -177,20 +178,20 @@ public override int GetHashCode()
                 return GetType().GetHashCode();
         }
 
-        public static Delegate Combine(Delegate a, Delegate b)
+        public static Delegate? Combine(Delegate? a, Delegate? b)
         {
-            if ((object)a == null) // cast to object for a more efficient test
+            if ((object?)a == null) // cast to object for a more efficient test
                 return b;
 
             return a.CombineImpl(b);
         }
 
-        public static Delegate Combine(params Delegate[] delegates)
+        public static Delegate? Combine(params Delegate[]? delegates)
```

Comment:
Shouldn't the parameter here be `params Delegate?[]? delegates`?  A null array is accepted, and null elements are accepted.

Reply:
Yep, still getting used to how this works.